### PR TITLE
Spelling fixes for contributing section

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: docs/contributing/javascript_guidelines.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Write JavaScript according to the `Airbnb Styleguide <http://github.com/airbnb/j
 -  Use soft-tabs with a four space indent. Spaces are the only way to
    guarantee code renders the same in any person's environment.
 -  We accept ``snake_case`` in object properties, such as
-   ``ajaxResponse.page_title``, however camelCase or UPPER_CASE should be used
+   ``ajaxResponse.page_title``, however ``camelCase`` or ``UPPER_CAS`` should be used
```
Comment (tomdyson) [inline docs/contributing/javascript_guidelines.rst]: CAS -> CASE

File: docs/contributing/javascript_guidelines.rst [hunk 2]
```diff
@@ -25,20 +25,16 @@ Ensure project dependencies are installed by running ``npm install``
...
 
 ``npm run format:js``
 
 This will perform safe edits to conform your JS code to the styleguide.
-It won't touch the line-length, or convert quotemarks from double to single.
+It won't touch the line-length, or convert quotation marks from double to single.
 
 Run the linter after you've formatted the code to see what manual fixes
-you need to make to the codebase.
+you need to make to the code base.
```
Comment (tomdyson) [inline docs/contributing/javascript_guidelines.rst]: let's stick with codebase

File: docs/contributing/javascript_guidelines.rst (hunk 1 above)
Comment (kaedroho) [inline docs/contributing/javascript_guidelines.rst]: oops

**Comment by kaedroho** [comment]
out of date

**Pull request closed without merging**